pkg/azure/resources.go

```diff
@@ -33,8 +33,17 @@ import (
 )
 
 const (
-	virtualMachineTypeName = "Microsoft.Compute/virtualMachines"
-	managedClusterTypeName = "Microsoft.ContainerService/managedClusters"
+	virtualMachineTypeName        = "Microsoft.Compute/virtualMachines"
+	managedClusterTypeName        = "Microsoft.ContainerService/managedClusters"
```

Comment:
Hm, the previous ones were here since the rest of the code in this file needs to tell those two resources apart. These other resource types are not relevant to anything else in this file. Could this all go in a utils thing instead?

Reply:
Yea I'll throw them in `utils.go` instead